transaction/atomicTransactionComposer.go

```diff
@@ -101,6 +101,15 @@ type AddMethodCallParams struct {
 
 	// References of the boxes to be accessed by this method call.
 	BoxReferences []types.AppBoxReference
+
+	// If true, the transaction will be created with the Access field
+	UseAccess bool
+	// The asset holdings to be accessed by this method call.
+	// An empty address means the sender.
+	Holdings []types.AppHoldingRef
+	// The local states to be accessed by this method call.
+	// An empty address means the sender.
```

Comment:
```suggestion
	// The zero address means the sender.
```
Should we also say that 0 app means the app being called?